packages/@react-spectrum/overlays/src/OpenTransition.tsx

```diff
@@ -32,7 +32,8 @@ const OPEN_STATES = {
  */
 
 export function OpenTransition(
-  props: TransitionProps
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
honestly, we should just turn off this eslint rule entirely. we decided we aren't going to do this
but I'm ok with that being follow up elsewhere